### PR TITLE
Update Install-Under-CentOS_LEMP.md

### PR DESCRIPTION
**Pull request by TheWitness**

### CODE REVIEW

File: Install-Under-CentOS_LEMP.md [hunk 1]
```diff
@@ -204,25 +204,45 @@ during the installation.
...
+   join_buffer_size = 32M
+   innodb_file_per_table = ON
+   innodb_large_prefix = 1
+   innodb_buffer_pool_size = 250M
+   innodb_additional_mem_pool_size = 90M
+   innodb_flush_log_at_trx_commit = 2
    log-error                      = /var/log/mysql/mysql-error.log
    log-queries-not-using-indexes  = 1
    slow-query-log                 = 1
    slow-query-log-file            = /var/log/mysql/mysql-slow.log
```
Comment (netniV) [inline Install-Under-CentOS_LEMP.md]: We should make this consistent, either with no spacing or make them aligned.

File: Install-Under-CentOS_LEMP.md [hunk 2]
```diff
@@ -204,25 +204,45 @@ during the installation.
...
+   character_set_client=utf8mb4
+   character_set_server=utf8mb4
+   collation_server=utf8mb4_unicode_ci
+   innodb_buffer_pool_size=8G
+   innodb_doublewrite=OFF
+   tmp_table_size=500M
+   log-error                      = /var/log/mysql/mysql-error.log
+   log-queries-not-using-indexes  = 1
+   slow-query-log                 = 1
+   slow-query-log-file            = /var/log/mysql/mysql-slow.log
```
Comment (netniV) [inline Install-Under-CentOS_LEMP.md]: We should make this consistent, either with no spacing or make them aligned.

**Review by netniV — CHANGES_REQUESTED** [review]

File: Install-Under-CentOS_LEMP.md (hunk 1 above)
Comment (TheWitness) [reply Install-Under-CentOS_LEMP.md]: done.

**Comment by TheWitness** [comment]
I made the updates.  Should be good now.

**Review by netniV — APPROVED** [review]

**Pull request merged**